controlpanel/frontend/views/reset.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+from django.urls import reverse_lazy
+from django.contrib import messages
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic.edit import FormView
+from controlpanel.frontend.forms import ResetHomeDirectoryForm
+from controlpanel.frontend.consumers import start_background_task
+
+
+class ResetHome(LoginRequiredMixin, FormView):
+    """
+    A view to help automate the resetting of the user's home directory. To be
+    used when, for example, their conda or r-studio deployment gets into a bad
+    state. See the related helm chart (TBD) for the details of what actually
```

Comment:
Maybe worth adding a link in this comment to the chart, now that it's in the helm charts repo https://github.com/ministryofjustice/analytics-platform-helm-charts/blob/main/charts/reset-user-home/

I personally find these clues/crumbs useful in the code.

Reply:
Resolved in latest commit.